README.md

```diff
@@ -177,6 +177,16 @@ svr.set_error_handler([](const auto& req, auto& res) {
 });
 ```
 
+### Exception handler
+
```

Comment:
Could you explain the purpose of this method in one or a few sentences? This is to intercept `std::exception &e` from users' routing handlers, but not from cpp-httplib itself, right?